src/App.js

```diff
@@ -9,6 +9,7 @@ function App() {
         <Route path="/" element={<Home/> }/>
         <Route path="/product" element={<Product/> }/>
         <Route path="/cart" element={<Cart /> }/>
+        <Route path="/wishlist" element={<Wishlist/> }/>
       </Routes>
```

Comment:
The `<Routes>` needs to be wrapped inside a `<BrowserRouter>` .

Reply:
it is wrapped in index.js